spring-pulsar-test/src/main/java/org/springframework/pulsar/test/support/PulsarConsumerTestUtil.java

```diff
@@ -80,17 +89,11 @@ public ConditionsSpec<T> until(ConsumedMessagesCondition<T> condition) {
 		return this;
 	}
 
-	@Override
-	public ConditionsSpec<T> withSchema(Schema<T> schema) {
-		this.schema = schema;
-		return this;
-	}
-
 	@Override
 	public List<Message<T>> get() {
 		var messages = new ArrayList<Message<T>>();
 		try {
-			String subscriptionName = UUID.randomUUID() + "-test-consumer";
+			var subscriptionName = "test-consumer-%s".formatted(UUID.randomUUID());
```

Comment:
Swapped to suffix as it will be easier to debug later "test-consumer-lfkdlsajskdljdfsa" than "lfjdslfjfadl-test-consumer" (probably)